src/services/pocket-relayer.ts

```diff
@@ -442,20 +443,24 @@ export class PocketRelayer {
     )
 
     if (pocketSession instanceof Session) {
-      const { sessionKey } = pocketSession
-
-      this.sessionKey = sessionKey
-
-      const sessionCacheKey = `session-${sessionKey}`
+      // Start the relay timer
+      const relayStart = process.hrtime()
 
-      let syncCheckPromise: Promise<Node[]>
-      let syncCheckedNodes: Node[]
+      let nodes: Node[] = pocketSession.sessionNodes
 
-      let chainCheckPromise: Promise<Node[]>
-      let chainCheckedNodes: Node[]
+      // sessionKey = "blockchain and a hash of the all the nodes in this session, sorted by public key"
+      const sessionKey = `${blockchain}-${crypto
+        .createHash('sha256')
+        .update(
+          JSON.stringify(
+            nodes.sort((a, b) => (a.publicKey > b.publicKey ? 1 : b.publicKey > a.publicKey ? -1 : 0)),
+            (k, v) => (k !== 'publicKey' ? v : undefined)
+          )
+        )
+        .digest('hex')}`
```

Comment:
This returns a string, right? Just to make extra extra sure

Reply:
It did for the first 18 months it was in use in the codebase